packages/hash/frontend/src/lib/config.ts

```diff
@@ -1,11 +1 @@
-export const FRONTEND_URL = process.env.NEXT_PUBLIC_FRONTEND_URL
-  ? process.env.NEXT_PUBLIC_FRONTEND_URL
-  : process.env.NEXT_PUBLIC_VERCEL_URL
-  ? `https://${process.env.NEXT_PUBLIC_VERCEL_URL}`
-  : "http://localhost:3000";
-
-export const FRONTEND_DOMAIN = new URL(FRONTEND_URL).hostname;
-
-export const isUsingHttps = new URL(FRONTEND_URL).protocol === "https";
-
 export const isProduction = process.env.NEXT_PUBLIC_VERCEL_ENV === "production";
```

Comment:
It can, but it's also very specific to vercel. I think we need another way to identify this (perhaps with the node environment instead?). I didn't want to put this there as it has nothing to do with the backend and the backend has its own way to check for being in production mode. Do you have some thoughts on what we should do? Perhaps `process.env.NODE_ENV === 'production' ??  export const isProduction = process.env.NEXT_PUBLIC_VERCEL_ENV === "production"`?

Reply:
In an in-person chat w/ Ahmad realised that `NEXT_PUBLIC_VERCEL_ENV` is specific to the frontend, so should stay there for now.